neard/src/lib.rs

```diff
@@ -171,5 +166,5 @@ pub fn start_with_config(
 
     trace!(target: "diagnostic", key="log", "Starting NEAR node with diagnostic activated");
 
-    (client_actor, view_client)
+    (client_actor, view_client, vec![client_arbiter, arbiter])
```

Comment:
why do we need to keep the arbiters?

Reply:
we join them at the end, it seems like System::run blocks until stop signal and we want to wait for all threads to end, but not 100% sure it's needed